healthtools/config.py

```diff
@@ -24,7 +24,7 @@
 
 # Where we archive the data
 DATA_DIR = os.getcwd() + "/data/"
-TEST_DIR = DATA_DIR + "tests/"
+TEST_DIR = os.getcwd() + "/healthtools/tests/"
```

Comment:
@Gathondu yeah, we're saying the same thing. So wouldn't it therefore go to the local data folder then? Or actually wherever the `DATA_DIR` is pointed to?